src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -4,12 +4,16 @@
 
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using Xunit;
 
 namespace System.IO.Tests
 {
     public class FileInfo_GetSetTimes : InfoGetSetTimes<FileInfo>
     {
+        private static bool isHFS => new DriveInfo(Path.GetTempPath()).DriveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
I noticed DriveFormat does not work in AppContainer. I was going to put up a PR to fix that
https://github.com/dotnet/corefx/compare/master...danmosemsft:fix.for.uap?expand=1

I suggest waiting until that's in then pulling that into this PR. The branch isn't open yet anyway